test/threading.cpp

```diff
@@ -653,3 +653,25 @@ TEST(Threading, DISABLED_MemoryManagerStressTest)
     t.join();
   }
 }
+
+TEST(Threading, BoostCompute)
+{
+    cleanSlate(); // Clean up everything done so far
+
+    vector<std::thread> tests;
+
+    ASSERT_EQ(AF_SUCCESS, af_set_device(0));
+
+    for (int i=0; i<THREAD_COUNT; ++i) {
+        tests.emplace_back([] {
+            for (int k=0; k<100; ++k) {
+                array a = randu(100, 100);
+                array b = sort(a);
+                array c = sort(a, 1, false);
```

Comment:
Try to test only one function. either sort or sort dim

Reply:
👍